docs/dom-testing-library/api-debugging.mdx

```diff
@@ -51,6 +51,7 @@ It is defined as:
 
 ```typescript
 function prettyDOM(
+  filterNode?: (node: Node) => boolean,
```

Comment:
I believe that this is wrong.
By taking a look at the [type definition](https://github.com/eps1lon/dom-testing-library/blob/alpha/types/pretty-dom.d.ts), I think that `filterNode` is a property of `options`?

Reply:
Thanks! Hope this is clearer now: [`df64629` (#866)](https://github.com/testing-library/testing-library-docs/pull/866/commits/df64629dbd4ee518491439b4caa13c0e11446998)

Reply:
For me it is, thanks!